wgsl/.pr-preview.json

```diff
@@ -0,0 +1,7 @@
+{
+    "src_file": "index.bs",
+    "type": "bikeshed",
+    "params": {
+        "force": 1
```

Comment:
I don't think we want this. Errors are bad.

Reply:
I thought this file was just for preview of the PR? In which case, don't we want to force it so we can see the rendering?